recipes/panzi_portable_endian/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+import os
+from conans import ConanFile, tools
+
+
+class PanziPortableEndian(ConanFile):
+    name = "panzi_portable_endian"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gist.github.com/panzi/6856583"
+    description = "This provides the endian conversion functions form endian.h on Windows, Linux, *BSD, and Mac OS X"
+    topics = ("conan", "endian")
+    license = "Public Domain"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder)
```

Comment:
You may want to look into the new `strip_root` option it might make the license extraction easier

Reply:
This is exactly what I want. Thank you!